.github/workflows/checks.yml

```diff
@@ -72,29 +72,26 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Check code lints with Black
-        uses: psf/black@stable
+      - name: ruff formatter
+        uses: chartboost/ruff-action@v1
         with:
-          version: 23.12.0
+          args: "format --check"
```

Comment:
Good to switch out black. I think it would be neat if we could also drop flake8 completely in favor of ruff I think.

Reply:
Agreed. I have removed all flake8 setup now.